hubspot/serializers.py

```diff
@@ -88,7 +101,7 @@ def get_company(self, instance):
         if redemption:
             company = redemption.coupon_version.payment_version.company
             if company:
-                return company.id
+                return company.name
```

Comment:
Does hubspot associate companies by name or id?  My guess was id but I could be wrong.

Reply:
Not sure what you mean here. The ecommerce bridge doesn't support company associations.

Reply:
I was probably getting ahead of myself here, I vaguely recall discussing this with @pdpinch and I think the the conclusion was this could wait til later.  Should be possible via the [CRM Associations API](https://developers.hubspot.com/docs/methods/crm-associations/crm-associations-overview)

Reply:
Ah I see. I don't know how easily that can be combined with the ecommerce bridge, since it looks like those are going to be accessed by their hubspot `objectId`s not our local `integratorId`s